src/cmd/kubectl.rs

```diff
@@ -1239,3 +1239,14 @@ where
         envs,
     )
 }
+
+pub fn kubernetes_get_all_hpas<P>(kubernetes_config: P, envs: Vec<(&str, &str)>) -> Result<HPA, SimpleError>
```

Comment:
Please take a look on how other similar cmd works to select namespaces. It would be simpler for later usage in a single namespace. Thx